### ISSUE TITLE
Clicking the "Add Reaction" Button removes contents of Embeds

### ISSUE THREAD

**Issue by sol-repos** [kyza-betterdiscord/ShowAllMessageButtons#1]
Expected Behaviour: You click on the "Add Reaction" Button on a message and the menu for selecting an emote opens

Actual Behaviour: The menu opens, but if it is a bot message with an embed, certain contents of the embed disappear

Notes: This only happens when the plugin is activated and the message button on the top right is used, if you right click and then add a reaction this doesn't happen

Before:
![image](https://user-images.githubusercontent.com/61390294/121413835-60a1dd80-c966-11eb-905f-a3c96114717a.png)

After:
![image](https://user-images.githubusercontent.com/61390294/121413952-80390600-c966-11eb-9d14-17ec93f78a02.png)

**Comment by Krazytrain** [comment]
Yeah this is still happening to me too. Really annoying the message gets removed.

**Comment by xsmashx88x** [comment]
sucks this never got fixed
